### ISSUE TITLE
Cinema Director dependency

### ISSUE THREAD

**Issue by andybak** [googlevr/tilt-brush-toolkit#30]
It seems a bit strange to have a dependency on a paid asset store component. I guess the Unity Timeline was released after this functionality was developed. 

Have the devs looked into replacing Cinema Director with the built-in timeline?

**Comment by dubois** [comment]
I'm guessing this refers to the Examples/ folder? That's one-off code that's mostly unrelated to the core functionality of the Toolkit; we're more likely to remove it from the repo than to update it, honestly.

**Comment by Michaelwolf95** [comment]
Back in Summer 2017, I worked on a project using an early version of the tool-kit, and collaborated with the GoogleVR team as they were developing it. 

It's possible that some of the stuff I did ended up in the examples, and this sounds like something I made back then. I used Cinema Director to setup some sequences using the audio-reactive shaders, and, yes, the built-in timeline wasn't out yet.

If this is the case, then I could probably resolve the dependency for you guys pretty quickly.
